app/controllers/miq_ae_class_controller.rb

```diff
@@ -2453,6 +2453,10 @@ def git_refresh
     session[:changed] = true
 
     git_repo = MiqAeDomain.find(params[:id]).git_repository
+
+    git_based_domain_import_service.refresh(git_repo.id)
```

Comment:
I think this method should take the `git_repo` object as the parameter and not just the ID.

Also, the `refresh` method returns `task_results`.  Are there plans to use that somewhere?  If not could `refresh` return the updated git_repo object which would remove the need for the following `reload` call?